schema/project-level/project-schema.json

```diff
@@ -1998,6 +2008,76 @@
         }
       },
       "minProperties": 1
+    },
+    "Meeting": {
+      "title": "Meeting",
+      "description": "The details of a meeting.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "title": "ID",
+          "description": "A locally unique identifier for the meeting.",
+          "type": [
+            "string"
+          ]
+        },
+        "date": {
+          "title": "Date",
+          "description": "The date of the meeting.",
+          "type": [
+            "string"
+          ],
+          "format": "date-time"
+        },
+        "location": {
+          "title": "Location",
+          "description": "The location of the meeting.",
+          "$ref": "#/definitions/Address"
+        },
+        "numberOfParticipants": {
+          "title": "Participant count",
+          "description": "The number of people in attendance at the meeting.",
+          "type": "integer"
+        },
+        "publicOfficial": {
+          "title": "Public official",
+          "description": "A position held by a person (or advisor to a person) with decision-making powers, who is elected, appointed or employed within the executive or legislative branches of power at national, sub-national, or supra-national levels; within private bodies performing public functions; and within public international organizations domiciled or operational in the country concerned.",
+          "$ref": "#/definitions/PublicOfficial"
+        }
+      },
+      "required": [
+        "id"
+      ],
+      "minProperties": 1
+    },
+    "PublicOfficial": {
+      "title": "Public official",
+      "description": "A position held by a person (or advisor to a person) with decision-making powers, who is elected, appointed or employed within the executive or legislative branches of power at national, sub-national, or supra-national levels; within private bodies performing public functions; and within public international organizations domiciled or operational in the country concerned.",
+      "type": "object",
+      "properties": {
+        "person": {
+          "title": "Person.",
+          "description": "The person.",
+          "type": "object",
+          "properties": {
+            "name": {
+              "title": "Name",
+              "description": "The full name of the person.",
+              "type": "string"
+            },
+            "organization": {
+              "title": "Organization",
+              "description": "The organization in respect of which the person is a public official.",
+              "$ref": "#/definitions/OrganizationReference"
```

Comment:
`.organization` should be a property of `PublicOfficial` rather than of `PublicOfficial.person` (`blank.json` and `example.json` will need updating too).

Reply:
oops, I had misread that, fixed now